packages/react-native-bridge/android/react-native-bridge/src/main/java/org/wordpress/mobile/WPAndroidGlue/GutenbergProps.kt

```diff
@@ -79,7 +79,7 @@ data class GutenbergProps @JvmOverloads constructor(
         private const val PROP_INITIAL_HTML_MODE_ENABLED = "initialHtmlModeEnabled"
         private const val PROP_POST_TYPE = "postType"
         private const val PROP_INITIAL_FEATURED_IMAGE_ID = "featuredImageId"
-        private const val PROP_LOCALE = "locale"
+        const val PROP_LOCALE = "locale"
```

Comment:
I had to make this public so it can be used in the Gutenberg demo app, eventually, we might consider change the rest too in case a future app client requires them 🤔 .